src/main/java/games/strategy/engine/framework/system/Memory.java

```diff
@@ -1,101 +1,50 @@
 package games.strategy.engine.framework.system;
 
-import java.util.Properties;
+import com.google.common.base.Strings;
 
 import games.strategy.debug.ClientLogger;
-import games.strategy.engine.framework.GameRunner;
+import games.strategy.engine.ClientContext;
+import games.strategy.engine.config.GameEngineProperty;
 
 public class Memory {
   public static final String TRIPLEA_MEMORY_SET = "triplea.memory.set";
-  private static final String TRIPLEA_MEMORY_XMX = "triplea.memory.Xmx";
-  private static final String TRIPLEA_MEMORY_USE_DEFAULT = "triplea.memory.useDefault";
 
 
-
-  public static void checkForMemoryXMX() {
+  public static boolean isMemoryXmxSet() {
     final String memSetString = System.getProperty(TRIPLEA_MEMORY_SET, "false");
     final boolean memSet = Boolean.parseBoolean(memSetString);
     // if we have already set the memory, then return.
     // (example: we used process runner to create a new triplea with a specific memory)
     if (memSet) {
-      return;
-    }
-    final Properties systemIni = GameRunner.getSystemIni();
-    if (useDefaultMaxMemory(systemIni)) {
-      return;
-    }
-    long xmx = getMaxMemoryFromSystemIniFileInMB(systemIni);
-    // if xmx less than zero, return (because it means we do not want to change it)
-    if (xmx <= 0) {
-      return;
+      return true;
     }
-    final int mb = 1024 * 1024;
-    xmx = xmx * mb;
-    final long currentMaxMemory = Runtime.getRuntime().maxMemory();
-    System.out.println("Current max memory: " + currentMaxMemory + ";  and new xmx should be: " + xmx);
-    final long diff = Math.abs(currentMaxMemory - xmx);
-    // Runtime.maxMemory is never accurate, and is usually off by 5% to 15%,
-    // so if our difference is less than 22% we should just ignore the difference
-    if (diff <= xmx * 0.22) {
-      return;
+
+    final String maxMemorySetting = ClientContext.propertyReader().readProperty(GameEngineProperty.MAX_MEMORY);
+    if (Strings.nullToEmpty(maxMemorySetting).isEmpty() || Long.parseLong(maxMemorySetting) < 32L) {
```

Comment:
Just to make sure I'm reading this correctly...  The minimum user max memory setting is 32 MiB?  Otherwise, we call them crazy and ignore what they provided?